nncf/onnx/graph/model_transformer.py

```diff
@@ -193,7 +200,8 @@ def _apply_quantizer_insertion_transformations(
         """
         self._added_target_edges = Counter()
         for transformation in transformations:
-            model = self._insert_quantizer_dequantizer(model, transformation)
+            children_node_mapping = get_children_node_mapping(model)
```

Comment:
I am not clear on why we should do this every iteration.

Reply:
Since this comment is not related to removing the ONNX graph, let's investigate this in a follow-up PR.

Reply:
After offline discussion we decided to make these changes in the follow up PR